cartography/intel/rapid7/vulnerabilities.py

```diff
@@ -0,0 +1,109 @@
+import logging
+from typing import Dict
+from typing import List
+
+import neo4j
+
+from .util import Rapid7Hosts
+from cartography.util import timeit
+
+logger = logging.getLogger(__name__)
+
+
+@timeit
+def sync_vulnerabilities(
```

Comment:
Because `Rapid7VulnerabilitiesByHost` is still "TBD", most of this file is still unreachable code, right?